webapp/src/components/chat/tabs/DocumentsTab.tsx

```diff
@@ -131,7 +131,7 @@ export const DocumentsTab: React.FC = () => {
                     type="file"
                     ref={documentFileRef}
                     style={{ display: 'none' }}
-                    accept=".txt,.pdf,.md,.jpg,.jpeg,.png,.tif,.tiff"
+                    accept=".txt,.pdf,.docx,.md,.jpg,.jpeg,.png,.tif,.tiff,.bmp,.gif"
```

Comment:
let's store this as a file-level constant in this file and then use it in `ChatInput.tsx`. no need to maintain this list twice

Reply:
Fair, but this aspect wasn't introduced in this change either.  I'll make a note to cycle back.  Honestly, this should be dynamic from the api...the whole thing is coincidental.